### ISSUE TITLE
Prefix inside `@supports` declaration

### ISSUE THREAD

**Issue by jacobbuck** [thysultan/stylis.js#34]
Input:

```css
/* @scope .foo */

input {
	@supports (appearance: none) {
		appearance: none;
	}
}
```

Expected result:

```css
@supports ((-webkit-appearance: none) or (-moz-appearance: none) or (appearance: none)) {
	.foo input {
		-webkit-appearance: none;
		-moz-appearance: none;
		appearance: none;
	}
}
```

Actual result:

```css
@supports (appearance: none) {
	.foo input {
		-webkit-appearance: none;
		-moz-appearance: none;
		appearance: none;
	}
}
```

**Comment by thysultan** [comment]
Interesting, i'm trying to understand why it would be used like that though since `@supports` suggests find granular control of feat support sniffing. In such a case what would be the difference between the following...

```css
input {
	@supports ((-webkit-appearance: none) or (-moz-appearance: none) or (appearance: none)) {
	      -webkit-appearance: none;
              -moz-appearance: none;
	      appearance: none;
	}
}
```

and

```css
input {
        -webkit-appearance: none;
        -moz-appearance: none;
	appearance: none;
}
```

**Comment by jacobbuck** [comment]
I'm basing this issue off what autoprefixer does. Also my example is simplified, it might look more like:

```
input {
	@supports ((-webkit-appearance: none) or (-moz-appearance: none) or (appearance: none)) {
	      -webkit-appearance: none;
              -moz-appearance: none;
	      appearance: none;
	      border-radius: 0;
	      box-shadow: inset 0 -2px 0 rgba(0, 0, 0, 0.2);
	      …
	}
```

**Comment by thysultan** [comment]
vendor prefixing the `@supports <functions>` might introduce issues if you want to check for a specific feature so it needs to retain what the user authors since it can't assume the intended use.

**Comment by jacobbuck** [comment]
However the contents of the `@supports` block is vendor prefixed. It seems inconsistent to do one and not the other.

**Comment by thysultan** [comment]
Interesting, i'm conflicted on what is the right approach, it might make sense to avoid prefixing a `@support` block since that assumes an explicit intent of authoring for a specific feature support. 

It could mimic autoprefixer though that prevents granular control of a specific property that defeats the use of `@supports` i.e different flex box implementations. 

On the other hand the current one is more a 50/50 approach.

**Comment by jacobbuck** [comment]
In autoprefixer the behaviour is enabled by default, but can be disabled either by `supports: false` option or `/* autoprefixer: off */` comment.

https://github.com/postcss/autoprefixer#disabling

**Comment by thysultan** [comment]
Patched in 3.2.13.
